product_docs/docs/pem/9/installing/dependencies.mdx

```diff
@@ -0,0 +1,141 @@
+| RHEL 8    | x86_64       | `edb-python310/edb-python310-mod-wsgi` | 3.10           | `/usr/libexec/edb-python310/bin/python3` |
+|           | ppc64le      | `edb-python310/edb-python310-mod-wsgi` | 3.10           | `/usr/libexec/edb-python310/bin/python3` |
+|           | s390x        | `edb-python310/edb-python310-mod-wsgi` | 3.10           | `/usr/libexec/edb-python310/bin/python3` |
+| RHEL 9    | x86_64       | `python39/python39-mod_wsgi`           | 3.9            | `/usr/bin/python3`                       |
+|           | ppc64le      | `python39/python39-mod_wsgi`           | 3.9            | `/usr/bin/python3`                       |
+|           | s390x        | `python39/python39-mod_wsgi`           | 3.9            | `/usr/bin/python3`                       |
+| SLES 12   | x86_64       | `edb-python39/edb-python39-mod-wsgi`   | 3.9            | `/usr/libexec/edb-python39/bin/python3`  |
+|           | ppc64le      | `edb-python39/edb-python39-mod-wsgi`   | 3.9            | `/usr/libexec/edb-python39/bin/python3`  |
+|           | s390x        | `edb-python39/edb-python39-mod-wsgi`   | 3.9            | `/usr/libexec/edb-python39/bin/python3`  |
+| SLES 15   | x86_64       | `edb-python310/edb-python310-mod-wsgi` | 3.10           | `/usr/libexec/edb-python310/bin/python3` |
+|           | ppc64le      | `edb-python310/edb-python310-mod-wsgi` | 3.10           | `/usr/libexec/edb-python310/bin/python3` |
+|           | s390x        | `edb-python310/edb-python310-mod-wsgi` | 3.10           | `/usr/libexec/edb-python310/bin/python3` |
+| Ubuntu 20 | amd64        | `edb-python310/edb-python310-mod-wsgi` | 3.10           | `/usr/libexec/edb-python310/bin/python3` |
+| Ubuntu 22 | amd64        | `python310//libapache2-mod-wsgi-py3`   | 3.10           | `/usr/bin/python3`                       |
+| Debian 10 | amd64        | `edb-python310/edb-python310-mod-wsgi` | 3.10           | `/usr/libexec/edb-python310/bin/python3` |
+
+## OpenSSL
+
+The PEM Server and Agent require OpenSSL.
+
+| Platform  | Architecture |  package-version  |
+|-----------|--------------|-------------------|
+| RHEL 7    | x86_64       | `openssl-1.0.2k`  |
+|           | ppc64le      | `openssl-1.0.2k`  |
+| RHEL 8    | x86_64       | `openssl-1.1.1k`  |
+|           | ppc64le      | `openssl-1.1.1k`  |
+|           | s390x        | `openssl-1.1.1k`  |
+| RHEL 9    | x86_64       | `openssl-3.0.7`   |
+|           | ppc64le      | `openssl-3.0.7`   |
+|           | s390x        | `openssl-3.0.7`   |
+| SLES 12   | x86_64       | `openssl-1.0.2p`  |
+|           | ppc64le      | `openssl-1.0.2p`  |
+|           | s390x        | `openssl-1.0.2p`  |
+| SLES 15   | x86_64       | `openssl-1.1.1l`  |
+|           | ppc64le      | `openssl-1.1.1l`  |
+|           | s390x        | `openssl-1.1.1l`  |
+| Ubuntu 20 | amd64        | `openssl-1.1.1f`  |
+| Ubuntu 22 | amd64        | `openssl-3.0.2`   |
+| Debian 10 | amd64        | `openssl-1.1.1n`  |
```

Comment:
I was thinking that we should not mention/list the dependencies, which are provided by the operating system (and - not built by us). It will be updated and maintained by the OS vendor.

@sonotley - WDYT?